helpdesk/views/staff.py

```diff
@@ -561,9 +561,8 @@ def update_ticket(request, ticket_id, public=False):
         comment=f.comment,
     )
 
-    if public and (f.comment or (
-        f.new_status in (Ticket.RESOLVED_STATUS,
-                         Ticket.CLOSED_STATUS))):
+    if f.comment or (
+       f.new_status in (Ticket.RESOLVED_STATUS, Ticket.CLOSED_STATUS)):
```

Comment:
Would this also be an issue with the upstream django-helpdesk?  The commit that moves `public` to be a requirement for e-mails does not mention this.